### ISSUE TITLE
Question about the voting module architecture

### ISSUE THREAD

**Issue by aphalak** [facebookresearch/votenet#6]
Hi,
According to the paper description of the voting module: 
``The voting module MLP has output sizes
of 256, 256, 259 for its fully connected layers. The last fully
connected layer does not have ReLU or BatchNorm.``

However, it seems that the FC layers in voting_module.py are replaced by convolutional layers : 
``self.conv1 = torch.nn.Conv1d(self.in_dim, self.in_dim, 1) ``
 `` self.conv2 = torch.nn.Conv1d(self.in_dim, self.in_dim, 1)  ``
`` self.conv3 = torch.nn.Conv1d(self.in_dim, (3+self.out_dim) * self.vote_factor, 1) ``

Is either known to perform better than the other? And how does the parameter sharing work in case of the Conv1d layers?
Thanks

**Comment by jytime** [comment]
actually here 1x1 Conv is just FC, i.e., MLP
